### DISCUSSION TITLE
version 0.2.4 released!

### DISCUSSION THREAD

**Discussion by AlexanderFengler** [lnccbrown/HSSM#586] [Announcements]
Version 0.2.4 is out! :trophy:

Here are some of the important new additions:

- We updated the old and added a few new tutorials, which you can find in the [documentation](https://lnccbrown.github.io/HSSM/) 
- Initial support for `variational inference` was added
- Improved, interface to give you full control over sampler initial values 
- Updated out backend to operate on the latest and greatest Bambi and PyMC versions
- Many other minor bug fixes which you can find in the [release notes](https://github.com/lnccbrown/HSSM/releases/tag/v0.2.4)

Thanks to @digicosmos86 and  @cpaniaguam for their contributions! :muscle:

**Comment by tigretigre** [comment]
When will this be updated on conda-forge?

**Reply by digicosmos86** [reply]
We are working on it. Should be done by EOD. Please stay tuned

**Reply by digicosmos86** [reply]
We have made a release on conda-forge for HSSM v0.2.4. The package is still building on CI and should be available in an hour or so. Please let us know if you run into any issues installing or using the new version.
